mods/lib/importPatterns.js

```diff
@@ -9,7 +9,7 @@ const _ = require('lodash');
 
 const impName = name => `(:<${name}>[a-zA-Z\\.\\$]+?)`;
 const modulePathRegex = `(:<dotPath>[.]+)?${impName('module')}`;
-const multipleModulesRegex = '(:<modules>[a-zA-Z\\.\\$]+[,]+[\s]*[a-zA-Z\\.\\$,\\s]+)';
+const multipleModulesRegex = '(:<modules>[a-zA-Z\\.\\$]+[,]+[s]*[a-zA-Z\\.\\$,\\s]+)';
```

Comment:
The eslint error was http://eslint.org/docs/rules/no-useless-escape I think...  The problem is you are escaping it in a string, not a regex.  If you want `\s` in the regex, you will need `\\s` in the string.  Just checked master; it is now `\\s`, looks like you caught it d: